go.mod

```diff
@@ -3,20 +3,18 @@ module sigs.k8s.io/kube-storage-version-migrator
 go 1.13
 
 require (
-	github.com/google/go-cmp v0.3.1 // indirect
-	github.com/googleapis/gnostic v0.2.0 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/onsi/ginkgo v1.11.0
-	github.com/prometheus/client_golang v1.0.0
+	github.com/prometheus/client_golang v1.7.1
```

Comment:
Why do we change prometheus and spf13 as well? Are they dependencies of k8s 1.19?